solidity/contracts/bridge/Fraud.sol

```diff
@@ -271,11 +272,18 @@ library Fraud {
     ///        Terminated state
     ///      - The `walletPublicKey` and `sighash` must identify an open fraud
     ///        challenge
+    ///      - The expression `keccak256(abi.encode(walletMembersIDs))` must
+    ///        be exactly the same as the hash stored under `membersIdsHash`
+    ///        for the given `walletID`. Those IDs are not directly stored
+    ///        in the contract for gas efficiency purposes but they can be
+    ///        read from appropriate `DkgResultSubmitted` and `DkgResultApproved`
+    ///        events.
```

Comment:
```suggestion
    ///        events of `WalletRegistry` contract.
```

Reply:
Done in 4f71462c58380aa3997fb91645eb80c3af0f86a1.